platform/core/src/defaults/hotkeyBindings.js

```diff
@@ -19,6 +19,11 @@ export default [
   { commandName: 'rotateViewportCW', label: 'Rotate Right', keys: ['r'] },
   { commandName: 'rotateViewportCCW', label: 'Rotate Left', keys: ['l'] },
   { commandName: 'invertViewport', label: 'Invert', keys: ['i'] },
+  {
+    commandName: 'cancelMeasurement',
+    label: 'Cancel Cornerstone Measurement',
```

Comment:
This is displayed in the hotkeys panel. Might be odd to specifically name `cornerstone`. The idea is that the command name is library agnostic, and each viewport can optionally implement the command